packages/devextreme/js/__internal/core/r1/config_context.ts

```diff
@@ -1,4 +1,4 @@
-import { createContext } from '@devextreme/runtime/inferno';
+import { createContext } from '@ts/core/r1/runtime/inferno/index';
```

Comment:
Personally i dont like the reexport without package (import from index)
I propose to add typescript path alias (or at least use file specific imports)

Reply:
Reexports are good, they allow us to make some files inside a folder private
E.g. you can create a lot of exports inside myfolder/utils.ts, but export only few of them in myfolder/index.ts